src/services/OfferService.js

```diff
@@ -33,48 +45,60 @@ export default class OfferService {
   getAllOffers (): Promise<void> {
     return Offer.find()
       .select('-_id -__v')
-      .populate({path: 'formData', select: '-_id -__v'})
+      .populate({ path: 'formData', select: '-_id -__v' })
       .exec()
   }
 
   getActiveOffers (city: string): Promise<void> {
     return Offer.find()
       .select('-_id -__v')
-      .where('city').equals(city)
-      .where('expirationDate').gt(Date.now())
-      .where('deleted').equals(false)
-      .populate({path: 'formData', select: '-_id -__v'})
+      .where('city')
+      .equals(city)
+      .where('expirationDate')
+      .gt(Date.now())
+      .where('deleted')
+      .equals(false)
+      .populate({ path: 'formData', select: '-_id -__v' })
```

Comment:
Is it forced, to have new lines when starting '.' (probably only when they overflow)?
I think the structure was better before.